### PR TITLE
Update fabric-features.md

### PR DESCRIPTION
**Pull request by Jonathan-Garriss**
Updating the feature list with AI Skills and some touch ups

# Thank you for contributing to Microsoft Fabric documentation

## Fill out these items before submitting your pull request:

If you are working internally at Microsoft:
**Provide a link to an Azure DevOps Boards work item that tracks this feature/update.**

>
>

**Who is your primary Skilling team contact?** \@mention them individually tag them and let them review the PR before signing off.

>
>

## For internal Microsoft contributors, check off these quality control items as you go

- [ ] 1. **Check the Acrolinx report:** Make sure your Acrolinx Total score is **above 80 minimum** (higher is better) and with no spelling issues. Acrolinx ensures we are providing consistent terminology and using an appropriate voice and tone, and helps with localization.

- [ ] 2. **Successful build with no warnings or suggestions**: Review the build status to make sure **all files are green** (Succeeded).

- [ ] 3. **Preview the pages:**: Click each **Preview URL** link to view the rendered HTML pages on the review.learn.microsoft.com site to check the formatting and alignment of the page. Scan the page for overall formatting, and look at the parts you edited in detail.

- [ ] 4. **Check the Table of Contents:** If you are adding a new markdown file, make sure it is linked from the table of contents.

- [ ] 5. **#sign-off to request PR review and merge**: Once the pull request is finalized and ready to be merged, indicate so by typing `#sign-off` in a new comment in the Pull Request. If you need to cancel that sign-off, type `#hold-off` instead. *Signing off means the document can be published at any time.* Note, this is a formatting and standards review, not a technical review.

## Merge and publish
- After you `#sign-off`, there is a separate PR Review team that will review the PR and describe any necessary feedback before merging. 
- The review team will use the comments section in the PR to provide feedback if changes are needed. Address any blocking issues and sign off again to request another review.
- Once all feedback is resolved, you can `#sign-off` again. The PR Review team reviews and merges the pull request into the specified branch (usually the main branch or a release- branch).
- From the main branch, the change is merged into the live branch several times a day to publish it to the public learn.microsoft.com site.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@KesemSharabi 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by KesemSharabi** [comment]
#sign-off

**Comment by Court72** [comment]
@Jonathan-Garriss, 

The article's author has signed off on these changes, but I see that there's [unresolved feedback](https://github.com/MicrosoftDocs/fabric-docs/pull/1286/files#r1847369289) that you might want to address. Can you confirm that this PR is ready to merge?

File: docs/enterprise/fabric-features.md [hunk 1]
```diff
@@ -19,14 +19,13 @@ The following table lists Fabric features according to SKU type. The table lists
...
 | [Power BI Autoscale](/power-bi/enterprise/service-premium-auto-scale)                                    | &#x274C;      | &#x2705; |
-| [Copilot](../get-started/copilot-fabric-overview.md)                                                     | F64 or higher | &#x2705; |
-| [Managed Private Endpoints](../security/security-managed-private-endpoints-overview.md)                  | All F SKUs or trial capacity | &#x274C; |
-| [Trusted workspace access](../security/security-trusted-workspace-access.md)                             | All F SKUs | &#x274C; |
+| [Copilot](../get-started/copilot-fabric-overview.md)                                                     | F64 or higher* | &#x2705; |
+| [Managed Private Endpoints](../security/security-managed-private-endpoints-overview.md)                  | F64 of higher | &#x274C; |
+| [Trusted workspace access](../security/security-trusted-workspace-access.md)                             | &#x2705;*     | &#x274C; |
+| [AI Skills](..fabric/data-science/concept-ai-skill.md)                                                   | &#x2705;      | &#x274C; |
```
Comment (Court72) [inline docs/enterprise/fabric-features.md:25]:
```suggestion
| [AI Skills](../fabric/data-science/concept-ai-skill.md)                                                   | &#x2705;      | &#x274C; |
```

File: docs/enterprise/fabric-features.md [hunk 2]
```diff
@@ -19,14 +19,13 @@ The following table lists Fabric features according to SKU type. The table lists
...
+| [Trusted workspace access](../security/security-trusted-workspace-access.md)                             | &#x2705;*     | &#x274C; |
+| [AI Skills](../fabric/data-science/concept-ai-skill.md)                                                   | &#x2705;      | &#x274C; |
 | [ARM APIs and Terraform](/azure/developer/terraform/overview-azapi-provider)                             | &#x2705;      | &#x274C; |
 | [Pause and resume your capacity](pause-resume.md)                                                        | &#x2705;      | &#x274C; |
 | [On-demand resizing](scale-capacity.md)                                                                  | &#x2705;      | &#x274C; |
-| [Bring your own key for Power BI](/power-bi/enterprise/service-encryption-byok)                          | &#x2705;      | &#x2705; |
-| [Private links](../security/security-private-links-overview.md)                                          | &#x2705;      | &#x2705; |
 | View Power BI items without a per user license                                                           | F64 or higher | &#x2705; |
```
Comment (Jak-MS) [inline docs/enterprise/fabric-features.md:30]:
```suggestion
\*Not supported in Fabric Trial capacity.
```

File: docs/enterprise/fabric-features.md [hunk 3]
```diff
@@ -19,16 +19,15 @@ The following table lists Fabric features according to SKU type. The table lists
...
-
+\*Not supported in Fabric Trial capacity.
```
Comment (Jak-MS) [inline docs/enterprise/fabric-features.md:30]:
```suggestion

\*Not supported in Fabric Trial capacity.

```

File: docs/enterprise/fabric-features.md [hunk 4]
```diff
@@ -19,16 +19,17 @@ The following table lists Fabric features according to SKU type. The table lists
...
 | [Power BI Autoscale](/power-bi/enterprise/service-premium-auto-scale)                                    | &#x274C;      | &#x2705; |
-| [Copilot](../get-started/copilot-fabric-overview.md)                                                     | F64 or higher | &#x2705; |
-| [Managed Private Endpoints](../security/security-managed-private-endpoints-overview.md)                  | All F SKUs or trial capacity | &#x274C; |
-| [Trusted workspace access](../security/security-trusted-workspace-access.md)                             | All F SKUs | &#x274C; |
+| [Copilot](../get-started/copilot-fabric-overview.md)                                                     | F64 or higher* | &#x2705; |
+| [Managed Private Endpoints](../security/security-managed-private-endpoints-overview.md)                  | F64 of higher | &#x274C; |
+| [Trusted workspace access](../security/security-trusted-workspace-access.md)                             | &#x2705;*     | &#x274C; |
+| [AI Skills](../fabric/data-science/concept-ai-skill.md)                                                   | &#x2705;      | &#x274C; |
```
Comment (Jak-MS) [inline docs/enterprise/fabric-features.md:25]:
```suggestion
| [AI Skills](../data-science/concept-ai-skill.md)                                                   | &#x2705;      | &#x274C; |
```

**Comment by Jak-MS** [comment]
@Jonathan-Garriss - I added the \* footnote and fixed the link to the concept-ai-skill.md article ... if this is ready to go, add a `#sign-off` or at-mention me in a comment. thanks.

**Comment by KesemSharabi** [comment]
@Jonathan-Garriss - it became too complicated to explain how to make the changes so I made them myself in the private repo, PR 6412.

#please-close

**Pull request closed without merging**